### DISCUSSION TITLE
Custom Device Attributes vs Tags

### DISCUSSION THREAD

**Discussion by ArumBlack** [opendcim/openDCIM#1546] [General]
Hello,

I am curious how people are using custom device attributes, Tags, why you chose one over the other, and what are some interesting or unusual things you have done with them?

Trying to get some inspiration to tackle some non standard things I need to track in our unique use case.

Thanks!

**Comment by samilliken** [comment]
Custom Device Attributes - let's say you are integrating openDCIM with some other software, like a CMDB.   There could be fields that you need to keep track of in order to sync, because the key field is different in the other software.   Tags are things that you set for a lot of things with the same value, while custom device attributes are common fields with unique information for each device.  At least, that's the intention.   Rarely do users continue to use things the way that we developers intend.

Scott
________________________________
From: ArumBlack ***@***.***>
Sent: Thursday, July 11, 2024 1:42 PM
To: opendcim/openDCIM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [opendcim/openDCIM] Custom Device Attributes vs Tags (Discussion #1546)


Hello,

I am curious how people are using custom device attributes, Tags, why you chose one over the other, and what are some interesting or unusual things you have done with them?

Trying to get some inspiration to tackle some non standard things I need to track in our unique use case.

Thanks!

**Reply by ArumBlack** [reply]
That is actually something I may need to do (integration with other software) I assume that is handled through the api? I will have to do some learning about how to utilize API's.

Another question, is it possible to enter a link that is clickable into the Custom attribute fields (or any field for that matter). for example, a network device with a web interface could have a link to open the interface in a browser?
